File: java/SR-RBD/src/ddb/msg/client/ConflictMessage.java
package ddb.msg.client;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.MessageType;

public class ConflictMessage extends ClientResponse {

	@Override
	public void fromBinary(DataInputStream s) throws IOException {
		// empty
	}

	@Override
	public MessageType getType() {
		return MessageType.CLIENT_CONFLICT;
	}

	@Override
	public void toBinary(DataOutputStream s) throws IOException {
		// empty
	}

}

File: trunk/C#/RBD/RBD/Msg/Client/ConflictMessage.cs
// +

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using RBD.Communication;

namespace RBD.Msg.Client
{
    class ConflictMessage : ClientResponse
    {
        override
        public void FromBinary(DataInputStream s) //throws IOException 
        {
            // empty
        }

        override
        public MessageType GetMessageType()
        {
            return MessageType.CLIENT_CONFLICT;
        }

        override
        public void ToBinary(DataOutputStream s) //throws IOException
        {
            // empty
        }
    }
}
